src/main/java/CodeBuilder.java

```diff
@@ -1131,6 +1164,31 @@ public ListBoxModel doFillSseAlgorithmItems() {
             return selections;
         }
 
+        public FormValidation doCheckMaxSleepTime(@QueryParameter String minSleepTime, @QueryParameter String maxSleepTime) {
```

Comment:
I encounter a NullPointerException in the FormValidation when I erase the values in "Maximum interval" or "Minimum interval". Can you add a null check for these values?